ambry-store/src/test/java/com.github.ambry.store/StorageManagerTest.java

```diff
@@ -575,6 +591,34 @@ public void skipStoppedStoresTest() throws Exception {
     shutdownAndAssertStoresInaccessible(storageManager, replicas);
   }
 
+  /**
+   * Test that tests that unrecognized directories are reported correctly
```

Comment:
Minor: redundant `tests that` in java doc

Reply:
Reworded